src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -408,27 +411,29 @@ pal::string_t get_deps_file(
 }
 
 /**
-* Given own location, FX location, app binary and specified --depsfile and probe paths
-*     return location that is expected to contain hostpolicy
+* Return location that is expected to contain hostpolicy
 */
-bool fx_muxer_t::resolve_hostpolicy_dir(host_mode_t mode,
+bool fx_muxer_t::resolve_hostpolicy_dir(
+    host_mode_t mode,
     const pal::string_t& own_dir,
-    const pal::string_t& fx_dir,
+    const std::vector<fx_definition_t*>& fx_definitions,
     const pal::string_t& app_candidate,
     const pal::string_t& specified_deps_file,
     const pal::string_t& specified_fx_version,
     const std::vector<pal::string_t>& probe_realpaths,
-    const runtime_config_t& config,
     pal::string_t* impl_dir)
 {
+    bool portable = fx_definitions[0]->get_runtime_config().get_portable();
```

Comment:
`fx_definitions[0]` being the "app" is kind of weird.  It isn't really a framework.

I'm not sure what to do about this.  Maybe not naming these things "fx"?  Wanted to get your thoughts on it.

Reply:
[0] is not a framework, but it contains the framework information (via runtimeconfig and deps) for the first framework, so kindof. I did add a get_app() helper method to fx_definitions_t to at least make it clear we're talking about the app's framework information.